frontend/src/Calendar/Agenda/Agenda.js

```diff
@@ -1,14 +1,50 @@
 import moment from 'moment';
 import PropTypes from 'prop-types';
 import React from 'react';
+import minNumberInArray from 'Utilities/Number/minNumberInArray';
 import AgendaEventConnector from './AgendaEventConnector';
 import styles from './Agenda.css';
 
 function Agenda(props) {
   const {
-    items
+    items,
+    start,
+    end
   } = props;
 
+  const startDateParsed = Date.parse(start);
+  const endDateParsed = Date.parse(end);
+
+  if (items) {
+    for (const i in items) {
+      if (items[i]) {
+        const cinemaDateParsed = Date.parse(items[i].inCinemas);
+        const digitalDateParsed = Date.parse(items[i].digitalRelease);
+        const physicalDateParsed = Date.parse(items[i].physicalRelease);
+        const dates = [];
+
+        if (cinemaDateParsed > 0 && cinemaDateParsed >= startDateParsed && cinemaDateParsed <= endDateParsed) {
+          dates.push(cinemaDateParsed);
+        }
+        if (digitalDateParsed > 0 && digitalDateParsed >= startDateParsed && digitalDateParsed <= endDateParsed) {
+          dates.push(digitalDateParsed);
+        }
+        if (physicalDateParsed > 0 && physicalDateParsed >= startDateParsed && physicalDateParsed <= endDateParsed) {
+          dates.push(physicalDateParsed);
+        }
+
+        items[i].sortDate = minNumberInArray(dates);
+        items[i].cinemaDateParsed = cinemaDateParsed;
+        items[i].digitalDateParsed = digitalDateParsed;
+        items[i].physicalDateParsed = physicalDateParsed;
+        items[i].startDateParsed = startDateParsed;
+        items[i].endDateParsed = endDateParsed;
```

Comment:
are startDateParsed  and endDateParsed ever used in AgendaEvent.js? If not they don't need to be added.